packages/presentation/lib/screens/home/widgets/shimmer_loader.dart

```diff
@@ -0,0 +1,79 @@
+import 'package:flutter/material.dart';
+import 'package:presentation/const.dart';
+import 'package:shimmer/shimmer.dart';
+
+class ShimmerLoader extends StatelessWidget {
+  static const gridItemsCount = 4;
+
+  const ShimmerLoader({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Expanded(
+      child: GridView.builder(
+        shrinkWrap: true,
+        itemCount: gridItemsCount,
+        gridDelegate: const SliverGridDelegateWithFixedCrossAxisCount(
+          crossAxisCount: 2,
+          childAspectRatio: HomeScreenSizes.gridChildAspecRatio,
+          crossAxisSpacing: AppSizes.size13,
+          mainAxisSpacing: HomeScreenSizes.gridViewMainAxisSpacing,
+        ),
+        itemBuilder: (context, index) {
+          return Column(
+            children: [
+              AspectRatio(
+                aspectRatio: 2 / 3,
+                child: Shimmer.fromColors(
```

Comment:
u can use shimmer as parent of grid view or column or even tabbar view section to apply to all containers below

Reply:
unfortunately for my implementation shimmer effect doesn't leave loading images from network